sources/packages/api/src/route-controllers/education-program/education-program.controller.ts

```diff
@@ -199,25 +202,25 @@ export class EducationProgramController extends BaseController {
       cipCode: educationProgram.cipCode,
       nocCode: educationProgram.nocCode,
       sabcCode: educationProgram.sabcCode,
-      approvalStatus: educationProgram.approvalStatus,
+      programStatus: educationProgram.programStatus,
       programIntensity: educationProgram.programIntensity,
       institutionProgramCode: educationProgram.institutionProgramCode,
-      submittedOn: educationProgram.submittedOn,
+      submittedOn: educationProgram.submittedDate,
       submittedBy: getUserFullName(educationProgram.submittedBy),
       effectiveEndDate: getISODateOnlyString(educationProgram.effectiveEndDate),
-      statusUpdatedOn: educationProgram.statusUpdatedOn,
+      assessedDate: educationProgram.assessedDate,
       // TODO: for now - program.effectiveEndDate is added by the ministry user
       // so, if program.effectiveEndDate is null/undefined, then
       // the program was auto approved, when institution submitted the
       // program, else the program was approved by ministry user.
       // ministry user uses IDIR. Program will always denied by
       // ministry user (i.e IDIR). Will need to update in future as
       // proper decision is taken
-      statusUpdatedBy:
+      assessedBy:
         educationProgram.effectiveEndDate ||
-        educationProgram.approvalStatus === ApprovalStatus.denied
-          ? getIDIRUserFullName(educationProgram.statusUpdatedBy)
-          : getUserFullName(educationProgram.statusUpdatedBy),
+        educationProgram.programStatus === ProgramStatus.Declined
```

Comment:
Can we please get rid of this logic and just return the `getUserFullName(educationProgram.assessedBy`?